libs/@blockprotocol/type-system/crate/src/ontology/entity_type/error.rs

```diff
@@ -33,5 +33,5 @@ pub enum ParseEntityTypeError {
 #[derive(Debug, PartialEq, Eq, Error)]
 pub enum MergeEntityTypeError {
     #[error("`{0}` is not contained in the `allOf` property of `{1}`")]
-    NotInAllOf(String, String),
+    NotInAllOf(VersionedUrl, VersionedUrl),
```

Comment:
do we maybe want to name the fields instead of using a tuple?

Reply:
Good point, made it a struct `{ child: _, parent: _}`